aiohttp/client.py

```diff
@@ -803,6 +803,7 @@ async def _ws_connect(
                                            autoclose,
                                            autoping,
                                            self._loop,
+                                           headers=real_headers,
```

Comment:
Are `ws.headers` for HTTP *client* headers sent to a server or *server* headers returned to the client back?